commands/motor/speedmodifiers/SetEnableableModifier.java

```diff
@@ -2,29 +2,34 @@
 
 
 import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.EnableableModifier;
-import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
 
-public class SetEnableableModifier extends Command {
+public class SetEnableableModifier extends CommandBase {
 	protected final EnableableModifier[] modifiers;
 	protected final boolean enable;
 
+
 	public SetEnableableModifier(boolean enable, EnableableModifier... modifiers) {
 		this.modifiers = modifiers;
 		this.enable = enable;
 		for (EnableableModifier modifier : modifiers) {
 			requires(modifier);
 		}
 	}
+	protected void requires(EnableableModifier modifier){
+
+	}
```

Comment:
Ok this seems wrong. You are calling the requires method, expecting it to do something, but the method is just empty. Did you accidentally add in your own requires method instead of changing the code to use the getRequirements method instead?